blackduck-security-task/test/unit/blackduck-security-task/bridge.spec.ts

```diff
@@ -683,7 +693,7 @@ describe("Download Bridge", () => {
 
         it("BRIDGECLI_DOWNLOAD_URL is defined and invalid for current os", async () => {
             Object.defineProperty(inputs, "BRIDGECLI_DOWNLOAD_URL", {
-                value: "https://repo.blackduck.com/bds-integrations-release/com/blackduck/integration/bridge-cli/0.2.57/bridge-cli-0.2.57-win64.zip",
+                value: "https://artifactory.internal.synopsys.com/artifactory/clops-local/clops.sig.synopsys.com/bridge/binaries/bridge-cli-bundle/2.9.2/bridge-cli-bundle-2.9.2-win64.zip",
```

Comment:
can we add some test cases for linux and windows as well?

Reply:
added unit tests for mac and linux